### ISSUE TITLE
tags for obejcts

### ISSUE THREAD

**Issue by anaismoller** [astrolabsoftware/fink-science-portal#510]
I love the tag idea for the portal. However I think that instead of repeating the information on the right (individual alert classification) that already has tags percentages, we should use these tags for other things.
- TNS reported (great and already implemented)
- TNS classified as X type (that would be super helpful!)

Thoughts?

Thanks!

**Comment by JulienPeloton** [comment]
Yes, I agree concerning the two bullet points! The TNS classification should appear though, I think this is a bug (I'm inspecting!). We could add any other external classification, but I do not know except TNS who is providing (via API) them?

However I would keep the repeated information for time being, as I had good feedback on it :-)

**Comment by JulienPeloton** [comment]
Bug found! Now reported (but not classified) in TNS would have the name, e.g.:
![Screenshot from 2023-10-16 09-22-38](https://github.com/astrolabsoftware/fink-science-portal/assets/20426972/2b8b0f6f-04f5-4b90-ae38-b4e677b23ebd)
and classified ones will have the name & classification, e.g.
![Screenshot from 2023-10-16 09-22-29](https://github.com/astrolabsoftware/fink-science-portal/assets/20426972/d5691ac0-6966-40b2-8ac7-92b5a11a75e8)

What do you think?
